docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,900 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #133.        chunk_start = query_idx + m
</pre><p>According to paper, this is to avoid trivial match. As discussed before, the paper considers<code>excl_zone=m</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/872/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>In the future, we should make sure to use the same exclusion zone as motifs (i.e., so that the discord discovered via DAMP will be the same as the classical <code>stumpy.stump</code> approach) </p>

Reply:
Right. later, I will bring back the `excl_zone` to make it behave like the discord discovery via `stumpy.stump`